check_tr64_fritz

```diff
@@ -244,310 +260,214 @@ queryResult=$(curl "https://${HOSTNAME}:${PORT}/upnp/control/${url}" \
 )
 
 if [[ "${queryResult}" == *"401 Unauthorized"* ]]; then
-	echo "ERROR - Wrong Password"
-	exit ${PLUGIN_ERROR}
+  echo "ERROR - Wrong Password"
+  exit ${PLUGIN_ERROR}
 fi
 
 if [ ${DEBUG} -eq 1 ]; then
-	echo "${queryResult}"
+  echo "${queryResult}"
 fi
 
 case ${FUNCTION} in
-	"status")
-		STATUS=$(find_xml_value "${queryResult}" "NewConnectionStatus")
+  "status")
+    STATUS=$(find_xml_value "${queryResult}" "NewConnectionStatus")
+    LASTERROR=$(find_xml_value "${queryResult}" "NewLastConnectionError")
+    CONTYPE=$(find_xml_value "${queryResult}" "NewConnectionType")
+    EXTERNALIP=$(find_xml_value "${queryResult}" "NewExternalIPAddress")
+    if [ -z "${STATUS}" ]; then
+      echo "UNKNOWN - Could not get a response | status=${PLUGIN_UNKNOWN}"
+      exit ${PLUGIN_UNKNOWN}
+    fi
+    outtext="${STATUS} \nLastError: ${LASTERROR} \nType: ${CONTYPE} \nExternalIP: ${EXTERNALIP}"
+    case ${STATUS} in
+      "Connected")
+        echo -e "OK - $outtext | status=${PLUGIN_OK}"
+        exit ${PLUGIN_OK}
+        ;;
+      "Connecting" | "Authenticating")
+        echo "WARNING - $outtext | status=${PLUGIN_WARNING}"
+        exit ${PLUGIN_WARNING}
+        ;;
+      *)
+        echo "ERROR - $outtext | status=${PLUGIN_ERROR}"
+        exit ${PLUGIN_ERROR}
+        ;;
+    esac
+    ;;
+  "lanlink")
+        echo "Result:${queryResult}"
+        # Dont know howto select Interface ?!
```

Comment:
For now please remove this and create an issue where we can discuss this feature and how to implement this.